pom.xml

```diff
@@ -45,6 +45,11 @@
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>com.github.ben-manes.caffeine</groupId>
+            <artifactId>caffeine</artifactId>
+            <version>2.6.2</version>
```

Comment:
We can bump this version, but it will require to bump caffeine version in GeoTrellis and to create two CQs.